### DISCUSSION TITLE
Artist Producer: need to make audiences aware of our upcoming multi-part series Elephant with livestream and in-person experiences.

### DISCUSSION THREAD

**Discussion by mariel-marshall** [pair-cg/general-contributions#86] [User Stories]
## 1. Motivating Story
I'm a member of the collective bluemouth inc. and we're currently producing a new show called Elephant. The production is a multi-part series, including x7 digital livestream events, followed by a live in-person show that audiences are invited to. This series finale will also be livestreamed. I need audiences to be aware of the full programming, and the different audience attendance modes for each.

# 2. Need:
I need audiences to be aware of the full experience programming, and the different audience attendance modes for each of the 8 individual events. Here is a breakdown of the information I would like to convey: 

**Overall production information:**

What is the name of the series?
Answer: Elephant
Proposed terminology: series_title
Instance of series_title: Elephant 

Competency Question: What kind of series is this? 
Answer: It's a multi-part series
Proposed terminology: _series type_
Instance of concept _series type_: multi part series

CQ: What style of performance is present?
Answer: It's an interdisciplinary performance
Proposed terminology: performance_type
Instance of performance_type: interdisciplinary performance

CQ: What kind of artistic discipline(s) are represented in the series?
Answer: film, dance, theatre, storytelling and music are the artistic disciplines represented.
Proposed terminology: artistic_discipline
Instances of artistic_discipline: film, dance, theatre, storytelling, music 

CQ: When does the series start?
answer: April 21, 2021 
Proposed terminology: series_start_date
Instance of series_start_date: April 21, 2021
 
CQ: When does the series end?
Answer: August 31, 2021
Proposed terminology: series_end_date
Instance of series_end_date: August 31, 2021

number_of_parts_in_series: 8 parts
name_of_producing_company: bluemouth inc.
experience_description: Over the last year and a half, the artists of bluemouth inc. have been creating and filming short works of dance, music, movement, voice-over, monologues and more in the development of a new piece entitled Elephant. Between April and July 2021, bluemouth inc. is releasing the seven episodes (10-15 minute shorts) on a bi-weekly schedule, with livestream events that include a performance screening and in-conversation artist talk-back. These works introduce audiences to the main characters and themes of Elephant. The episodes serve as a companion to the live event which will take place this August (exact date is TBD).
marketing_description: An invitation to join seven friends as they emerge from the dark forest of the past year and gather around the communal table to share their stories, strengths, dreams, and desires. Food for thought. A feast for the senses. Nourishment for the soul. Follow us on social media for the most up-to-date info! 

**Episode 1**
sub_experience_title: Lucy
series_part: episode 1
sub_experience_type: interdisciplinary performance screening and in-conversation artist talk-back with the creative team
tag-line: This pause in our history sets us back, slows us down, brings us outdoors, reminds us we are interconnected. Not just human to human, but to our natural world.
artistic_writing_to_accompany_experience: We are all Persephone, plunged into Hades, tricked into undervaluing our gifts, our creativity, our communities of support. We exchange them for ambition, commerce, success, individual recognition...
teaser_trailer_url: n/a
digital_short_url: [https://vimeo.com/541418307](https://vimeo.com/541418307)
date: April 21, 2021
time: 8:30pm ET
duration: 30-40 minutes
status: experience scheduled on time
audience_attendance_mode: remote access 
streaming_platform: vimeo
artistic_discipline: film, dance, storytelling
remote_format: Audio Video
remote_format_access_type: livestream
experience_location: online_location
online_location_url: [https://www.bluemouthinc.live/live-stream](https://www.bluemouthinc.live/live-stream)
description_of_experience_components: 
- Introduction from the company (live material)
- Screening of a 10-15 minute video (recorded material)
- In conversation Q&A with the creative team (live material with audience interaction)

sub_experience_name: Episode 2
sub_experience_title: Tony     
Date: May 5, 2021
...same as above
format follows for Episode #3- #7

**In-Person Series Finale**
sub_experience_name: Series Finale
sub_experience_title: Elephant
Date: August 2021 (TBD) - the general date is currently scheduled for August but the exact date has not yet been confirmed. 
status: experience pending 
artistic_disciplines: theatre, music, dance, video, projection
staging: site-specific
seating: picnic blankets provided or bring your own lawn chair

**In-person details:** 
audience_attendance_mode: in-person attendance (IRL in real life)
address: to be confirmed
experience_location: physical_experience_location
immersivity level: high (in-person audiences are fully immersed in the performance world)
interactivity level: high (audiences interact with performers, and can take on performative roles in the work)

**remote access details**
audience_attendance_mode: remote access 
remote_format: Audio Video
remote_format_access_type: livestream
experience_location: online_location
streaming_platform: vimeo
registration: no advanced registration required 
ticket cost: free

# 3. Action:

I publish this content on our website so that it can be accessed by search engines, digital assistants, recommenders, audiences, researchers and the media.

# 4. Desired Outcome:

I make search engines, digital assistants, recommenders, audiences, researchers and the media aware of the work we are doing, and they are clear on how to attend it.

# 5. Perspective:

Artist-producer

# 6. Stakeholder Group:

Audiences
Artists

# 6. Additional info:

url: [https://www.bluemouthinc.live/](url)

**Comment by drosu** [comment]
Hi @mariel-marshall, 
I have a few questions:
1. Would it be accurate to describe the series as a "multi-part" series? (YES!) 
1. does series_dates refer to the "start date" and the "end date" for the series ? (YES!)
2. does number_of_events_in_series refer to the number of "parts" in the series? (YES!)
3. what is the difference between sub_experience_name and sub_experience_title ? (well, we say Episode 1, and then in next frame the title of it, Lucy, so I was trying to capture that)
4. Is a sub-experience intended to have a fixed duration? No, they don't have a set duration, depends on how long the Q&A goes.
4. Are potential audience members expected to infer that there is an audio component in addition to the visual component? (the remote_format is listed as video). (YES!)
Thanks!
